lib/ahoy/intercom/store.rb

```diff
@@ -0,0 +1,41 @@
+require 'ahoy_matey'
+require 'intercom'
+
+module Ahoy
+  module Intercom
+    class Store < ::Ahoy::Stores::ActiveRecordStore
```

Comment:
Why does it inherit from ActiveRecordStore? What if end user doesn't want to use ActiveRecord store and prefers to store events _only_ in Intercom?